### PR TITLE
update main readme for supporting EAST model; improve readme_cn of EAST; add readme_en of EAST

### PR DESCRIPTION
**Pull request by tonytonglt**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Please refer to [MindOCR Online Inference](tools/infer/text/README.md) for detai
 - [x] [DBNet](configs/det/dbnet/README.md) (AAAI'2020) 
 - [x] [DBNet++](configs/det/dbnet/README.md) (TPAMI'2022)
 - [x] [PSENet](configs/det/psenet/README.md) (CVPR'2019)
-- [ ] [EAST](https://arxiv.org/abs/1704.03155)(CVPR'2017) [coming soon]
+- [x] [EAST](https://arxiv.org/abs/1704.03155)(CVPR'2017)
```
Comment (SamitHuang) [inline README.md:119]: link to configs/det/east/README.md

Comment (tonytonglt) [reply README.md]: fixed

**Review by SamitHuang — APPROVED** [review]

File: configs/det/east/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+|------------------|----------------|-------------|------------|------------|---------------|-------------|-------------|-----------|----------------------------|------------------------------------------------------------------------------------------------------------|
+| EAST             | D910x8-MS1.9-G | ResNet-50   | ImageNet   | 82.23%     | 87.68%        | 84.87%      | 1.6 s/epoch | 625 img/s | [yaml](east_r50_icdar15.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/east/east_resnet50_ic15-7262e359.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/east/east_resnet50_ic15-7262e359-5f05cd42.mindir) |
+| EAST (PaddleOCR) | -              | ResNet50_vd | ImageNet   | 81.36%     | 88.71%        | 84.88%      | -           | -         | -                          | -                                                                                                          |
```
[the remark this answers is not in the document]
Comment (tonytonglt) [reply configs/det/east/README.md]: deleted

File: configs/det/east/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+</div>
+
+#### Notes：
+- Context：Training context denoted as {device}x{pieces}-{MS version}{MS mode}, where mindspore mode can be G - graph mode or F - pynative mode with ms function. For example, D910x8-G is for training on 8 pieces of Ascend 910 NPU using graph mode.
```
[the remark this answers is not in the document]
Comment (tonytonglt) [reply configs/det/east/README.md]: fixed

**Pull request merged**